motoman_msgs/srv/SelectTool.srv

```diff
@@ -0,0 +1,132 @@
+
+# Change the active tool file on the controller.
```

Comment:
I like the level of documentation here, but it might be more accessible to an average user in a wiki page or README

Reply:
True, but that would increase the distance between the `.srv` and its documentation.

I also estimate it's less likely for a ROS developer to lookup a wiki when looking for message documentation than looking at the `.srv` itself, either directly or through `rossrv show`.

Additionally, if `motoman_msgs` ever gets released, the buildfarm would generate HTML documentation based on this "for free".

We could add a reference to a readme though.

Reply:
Good points. I'm fine with leaving it in the `srv` then.